code/api/src/main/scala/com/avast/datadog4s/api/event/AlertType.scala

```diff
@@ -0,0 +1,5 @@
+package com.avast.datadog4s.api.event
+
+object AlertType extends Enumeration {
+  val Error, Warning, Info, Success = Value
```

Comment:
please use `sealed trait` and `case object` to represent enums - scala `Enumeration` suffers from not having exhaustive compile time check iirc